shm/Island.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include "Coordinates.h"
```

Comment:
I think there should be a newline between pragma and include.

Reply:
why? is that good practice?